### DISCUSSION TITLE
How to start BackRest.exe on Windows and hide the console window?

### DISCUSSION THREAD

**Discussion by berti34** [garethgeorge/backrest#289] [Q&A]
How can I start BackRest.exe in background on Windows and hide the console window completely?

**Comment by scialo** [comment]
![immagine](https://github.com/garethgeorge/backrest/assets/22222042/0f357f81-ca52-4bda-85bb-03b13aa85c21)


![immagine](https://github.com/garethgeorge/backrest/assets/22222042/9434c1fd-45b0-45ec-9bcd-6a3e7279187e)


![immagine](https://github.com/garethgeorge/backrest/assets/22222042/78be6387-9a07-4430-ac95-e01847eaf68c)

**Reply by berti34** [reply]
Thanks a lot!
